src/main/java/com/minecolonies/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -828,9 +828,9 @@ private static ItemStack handleKeepX(@NotNull Map<ItemStorage, Integer> keptX,
      */
     private static boolean keptEnough(@NotNull Map<ItemStorage, Integer> kept, @NotNull Map<ItemStorage, Integer> keep, @NotNull ItemStack stack)
```

Comment:
@Kostronor I understand why, just not why the way we are doing it the way we are.

@Raycoms I will have to look at it again, I definitely would recommend a variable name change, maybe have and want, or kept and want, haveKept and wantToKeep. Something more than just a single letter difference in the middle of the word :)